### PR TITLE
persona valora customization

### LINKED ISSUE #1156: [M1] Style Persona widget with Valora design patterns
The Persona widget that will be embedded into Valora comes with customizable theming. The styling of the Persona widget should resemble Valora styling as closely as possible.

Persona theming docs: https://docs.withpersona.com/docs/react-native-sdk-v2-theming
Designs: https://www.figma.com/file/rFsQPU838CfakbNTOjHaxb/KYC?node-id=349%3A7609

AC:
- [ ] Persona widget styling matches our designs as closely as possible

### PR DESCRIPTION
**Pull request by lisaqyli**
### Description
This PR styles persona flow with a Valora touch.  See screenshots below on what it looks like. 
I failed to provide custom font on Android, if anyone reviewing this PR knows how to do it, would love to learn some tricks.
I'm also having build issue on iOS (apparently it's an issue with my machine not my branch), hence the lack of iOS screenshots, I'm planning to get my iOS build working and do a final verification before merging this PR

### Tested
manually tested

### Related issues
- Fixes #1156 

259988915_4873715992680586_4386038747511032186_n

260692576_287171723347701_5926472412340243340_n

260074775_324961699103043_836020520981946847_n

260284139_630794127955019_2546921526241804134_n

260699554_4348849045225546_3948794866310632064_n

260048279_423155276182827_1533736773107574348_n

260024713_424458385910489_3480914069284171792_n

### CODE REVIEW

File: packages/mobile/android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,3 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-</resources>
\ No newline at end of file
+  <string name="persona_inquiry_start_title">Valora would like to verify your identity</string>
+  <!-- todo: Lisa need to fix the string below -->
```
Comment (cajubelt) [inline packages/mobile/android/app/src/main/res/values/strings.xml:4]: todo

**Review by cajubelt — COMMENTED** [review]
looks like there's a hanging todo

**Review by jh2oman — APPROVED** [review]

**Review by cajubelt — APPROVED** [review]
nice!!

**Review by jophish — APPROVED** [review]

**Comment by MuckT** [comment]
@lisaqyli it looks like the `exchange.spec.ts` tests are failing on Android:
```
FAIL e2e/src/Exchange.spec.js (787.267 s)
  Given Exchange
    When CELO
      ✕ Then Buy CELO (46697 ms)
      ✕ Then Sell CELO (73142 ms)
      ✕ Then Withdraw CELO (50785 ms)
      ○ skipped Then Sell Minimum CELO
```

I'd recommend working taking a look at these flows in dev mode and check for leaks with Leak Canary.

beforeEachFailure

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified this issue using iOS Alfa jores build V1.23.0 & updated the status in Gsheet
**NOTE:** Not verified on Android platform, as for android platform Latest Alfa jores build V1.22.4 is displayed.

**Sheet Link;**
https://docs.google.com/spreadsheets/d/15Z15zu4roT6XFD1dbi9__c3Q6aQZpIiIZka1rKVrCgw/edit#gid=1915406409

**New Bug:** [Link](https://github.com/valora-inc/wallet/issues/1630)
Devices: iPhone XR(14.6), iPhone 13 mini (15.1.1)
CC: @lisaqyli
Thanks..!!

**Pull request merged**